### DISCUSSION TITLE
When will 4.0 be released?  Looking for 'L' support in DOM

### DISCUSSION THREAD

**Discussion by brandonyoung** [quartznet/quartznet#2265] [General]
Looks like support was added in 2023 but 4.0 has been running/open for a few years.  Is there any chance this sees the light of day anytime soon?

**Comment by lahma** [comment]
No planned release schedule, scheduling is hard.

You should be able to get v4 previews from feedz NuGet feed documented in main readme though. Always good to have people trying out the bits.

**Reply by mgernand** [reply]
Hi @lahma,

is there a list of open issues that are to be finished to be able to release v4.0? I'd like to help but can not find anything that would allow me to choose a task and implement it. 

Is the current v4.0 codebase found in the main branch, or in the v4 branch?

Cheers, Matt

**Reply by jafin** [reply]
@mgernand  v4 Roadmap https://github.com/quartznet/quartznet/issues/988  main is v4
